Tasks/Kulik/Task1/1.py

```diff
@@ -0,0 +1,5 @@
+a = int(input("Deposit amount:\n")) 
+b = int(input('Years amount:\n'))
+c = int(input('Procent:\n'))
+x=a*((1+b/100/12)**(c*12))
```

Comment:
By adapting these values for the **int** type and using them in the math expression you get the **float** number as answer, which tends to accumulate some error, it's better to use **decimal** for money-related operations, -10 points for the logical mistake